package.json

```diff
@@ -83,5 +83,10 @@
   "prettier": {
     "semi": false,
     "singleQuote": true
+  },
+  "pnpm": {
+    "overrides": {
+      "@nuxtjs/storybook": "workspace:*"
+    }
```

Comment:
Adds local support for our local package, while supporting the StackBlitz configs in the `examples/*` directories.